### PR TITLE
basics/exceptions: neutral style

### PR DESCRIPTION
**Pull request by wilzbach**

### CODE REVIEW

File: basics/exceptions.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ finally
 }
 ```
 
-Remember that the scope guard is usually a better solution to the `try-finally`
+Remember that a scope guard is usually a better solution to the `try-finally`
```
Comment (wilzbach) [inline basics/exceptions.md]: do we mention scope guards before? I couldn't find it :/

Comment (wilzbach) [inline basics/exceptions.md]: Btw I feel more and more that we should move away from the traditional order (basics  / gems) as we hide all the cool part from the reader for a long long time (and they usually bounce before)

Comment (JackStouffer) [inline basics/exceptions.md]: Link to the scope guard section?

File: basics/exceptions.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ finally
 }
 ```
 
-Remember that the [scope guard](https://dlang.org/spec/statement.html#ScopeGuardStatement) is usually a better solution to the `try-finally`
+Remember that the [scope guard](gems/scope-guards) is usually a better solution to the `try-finally`
```
Comment (wilzbach) [inline basics/exceptions.md]: A bit of advertisement and thus link to the respective section in the DLang Tour

Comment (wilzbach) [inline basics/exceptions.md]: Note that the link to the gems scetion will be "broken" (the fix is https://github.com/stonemaster/dlang-tour/pull/472)

**Comment by JackStouffer** [comment]
Is this ready to be merged, or do we have to wait on the link fix?

**Comment by wilzbach** [comment]
> Is this ready to be merged, or do we have to wait on the link fix?

Yes, link PR will hopefully be merged soon.

**Pull request merged**